src/main/java/org/jenkinsci/plugins/github_branch_source/ForkPullRequestDiscoveryTrait.java

```diff
@@ -85,26 +85,34 @@
     /**
      * Constructor for stapler.
      *
-     * @param strategyId the strategy id.
-     * @param trust      the authority to use.
+     * @param strategyId
+     *            the strategy id.
+     * @param trust
+     *            the authority to use.
      */
     @DataBoundConstructor
-    public ForkPullRequestDiscoveryTrait(int strategyId,
-                                         @NonNull SCMHeadAuthority<? super GitHubSCMSourceRequest, ? extends ChangeRequestSCMHead2, ? extends SCMRevision> trust) {
+    public ForkPullRequestDiscoveryTrait(
+            int strategyId,
+            @NonNull SCMHeadAuthority<? super GitHubSCMSourceRequest, ? extends ChangeRequestSCMHead2, ? extends SCMRevision> trust) {
         this.strategyId = strategyId;
         this.trust = trust;
     }
 
     /**
      * Constructor for programmatic instantiation.
      *
-     * @param strategies the {@link ChangeRequestCheckoutStrategy} instances.
-     * @param trust      the authority.
+     * @param strategies
+     *            the {@link ChangeRequestCheckoutStrategy} instances.
+     * @param trust
+     *            the authority.
      */
-    public ForkPullRequestDiscoveryTrait(@NonNull Set<ChangeRequestCheckoutStrategy> strategies,
-                                         @NonNull SCMHeadAuthority<? super GitHubSCMSourceRequest, ? extends ChangeRequestSCMHead2, ? extends SCMRevision> trust) {
-        this((strategies.contains(ChangeRequestCheckoutStrategy.MERGE) ? MERGE : NONE)
-                + (strategies.contains(ChangeRequestCheckoutStrategy.HEAD) ? HEAD : NONE), trust);
+    public ForkPullRequestDiscoveryTrait(
+            @NonNull Set<ChangeRequestCheckoutStrategy> strategies,
+            @NonNull SCMHeadAuthority<? super GitHubSCMSourceRequest, ? extends ChangeRequestSCMHead2, ? extends SCMRevision> trust) {
+        this(
+                (strategies.contains(ChangeRequestCheckoutStrategy.MERGE) ? MERGE : NONE)
+                        + (strategies.contains(ChangeRequestCheckoutStrategy.HEAD) ? HEAD : NONE),
+                trust);
```

Comment:
worse than before IMO

Reply:
@jtnord 
Agreed, but I'm not sure the code itself was great to begin with.